test/common/dashboards_data.go

```diff
@@ -370,7 +371,7 @@ func getPrometheusQueryResult(output string) ([]prometheusQueryResult, error) {
 	}
 
 	if len(queryResponse.Data.Result) == 0 {
-		return nil, fmt.Errorf("no result")
+		return nil, nil
```

Comment:
🤔 hmm, why has this changed?

Reply:
The change here is because the test will eroor if no pod is found in a namespace. Because we've change the earlier query from getting namespaces by middleware monitoring label to `redhat-rhoam-*` we no also pick up `redhat-rhoam-monitoring and  redhat-rhoam-middleware-monitoring-federate` which don't have pods in them. So the pod query fails on no pod but which fails the test. I was thinking we could add two functions, one which errors on no value and one which doesn't and use them based on the situation. The function is called 3 times.